### DISCUSSION TITLE
纯Vue工程使用Taro组件

### DISCUSSION THREAD

**Discussion by xyyyzzz** [NervJS/taro#16810] [Q & A]
是否可以在纯Vue项目中使用Taro组件（只用适配h5端）？一定要用@tarojs/cli搭建的taro工程吗？

**Comment by hanjin001** [comment]
为啥要用 `taro` 组件呢，有 [vant](https://github.com/youzan/vant) 等移动端的组件库来实现功能呀。

`@tarojs/cli` 主要是生成小程序/H5项目模板的，使用的框架是 `Taro`。

**Reply by xyyyzzz** [reply]
我们做的项目是基于taro框架的组件库，项目中用了taro的原生组件，用native-components打包组件库的话只能在taro项目中用，不能在纯h5环境用
